recipes/tjpcov/meta.yaml

```diff
@@ -0,0 +1,109 @@
+{% set name = "tjpcov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tjpcov-{{ version }}.tar.gz
+  sha256: fd4cd437105521e11c7f977d522cdfae75fed6d7e0c06afa599cc7eec8d7a4eb
+
+build:
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - pip
+  run: 
+    - python >=3.8
+    - scipy
+    - numpy
+    - jinja2
+    - pyyaml
+    - pyccl >=3.0.0
+    - sacc >=0.12
+    - camb
+    - healpy
+    - h5py
+
+test:
+  imports:
+    - tjpcov
+
+outputs:
```

Comment:
I'm not sure those outputs are working as you expected.

Reply:
@ocefpaf Can you elaborate?  I may need a little help here, first conda recipe I've created.  Thank you!

Reply:
@ocefpaf Thanks for the guidance on this, I fixed my other outputs to depend on the main package as in the example :+1: